dev/com.ibm.ws.jpa.tests.spec10.injection.mdb_fat/fat/src/com/ibm/ws/jpa/tests/spec10/injection/mdb/JPA10Injection_MDB.java

```diff
@@ -0,0 +1,319 @@
+import com.ibm.websphere.simplicity.config.ConfigElementList;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.anoovrd.DFIPkgYesInhAnoOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.anoovrd.DFIPriYesInhAnoOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.anoovrd.DFIProYesInhAnoOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.anoovrd.DFIPubYesInhAnoOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.ddovrd.DFIPkgYesInhDDOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.ddovrd.DFIPriYesInhDDOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.ddovrd.DFIProYesInhDDOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.ddovrd.DFIPubYesInhDDOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.noovrd.DFIPkgNoInhMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.noovrd.DFIPriNoInhMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.noovrd.DFIProNoInhMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dfi.noovrd.DFIPubNoInhMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.anoovrd.DMIPkgYesInhAnoOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.anoovrd.DMIPriYesInhAnoOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.anoovrd.DMIProYesInhAnoOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.anoovrd.DMIPubYesInhAnoOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.ddovrd.DMIPkgYesInhDDOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.ddovrd.DMIPriYesInhDDOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.ddovrd.DMIProYesInhDDOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.ddovrd.DMIPubYesInhDDOvrdMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.noovrd.DMIPkgNoInhMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.noovrd.DMIPriNoInhMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.noovrd.DMIProNoInhMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.dmi.noovrd.DMIPubNoInhMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.jndi.AnnotatedJNDIMDBServlet;
+import com.ibm.ws.jpa.fvt.injection.tests.mdb.jndi.DDJNDIMDBServlet;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.annotation.TestServlets;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.database.container.DatabaseContainerType;
+import componenttest.topology.database.container.DatabaseContainerUtil;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.PrivHelper;
+
+@RunWith(FATRunner.class)
+@Mode(TestMode.FULL)
```

Comment:
Per above comment, would it be better to remove this "@Mode(TestMode.FULL)", and instead add ".fullFATOnly()" to all the above RepeatActions except one.  In this way the test will run in LITE mode for at least one action.

Or, perhaps add FULL mode to individual test methods, so at least some could run in LITE.